roles/ceph-rgw/tasks/multisite/secondary.yml

```diff
@@ -1,4 +1,12 @@
 ---
+- name: ensure connection to primary cluster
+  uri:
+    url: "{{ item.endpoint }}"
+  delegate_to: "{{ groups[mon_group_name][0] }}"
+  run_once: true
```

Comment:
related to my previous comment but if we need to test the connection from the rgw nodes, should we test this from all nodes instead of only one ?

Reply:
Just added a commit for this. In my commit each rgw (which is zonesecondary) does uri over rgw endpoints in it's `rgw_instances`.